math/median/median.go

```diff
@@ -0,0 +1,31 @@
+// author(s) [jo3zeph](https://github.com/jo3zeph)
+// description: Find the median from a set of values
+// see median_test.go
+
+package median
+
+import (
+	"sort"
+)
+
+
+func median(values []float64) float64 {
```

Comment:
Please consider capitalising this function and using generics to make it more usable outside the context of `float64`.

If you don't capitalise the function then the automatic documentation tool that we use will not be able to pick that up.